modules/candidate_list/templates/menu_candidate_list.tpl

```diff
@@ -1,4 +1,11 @@
-<div class="row">
+{literal}
+<script type="text/javascript">
+    $(document).ready(function(){
+        $("#cand").DynamicTable({ "freezeColumn" : "pscid" });
```

Comment:
Can you document somewhere (maybe at the top of the js file?) what the parameter expected is?

I'm assuming it's the name of the header, but it should be explicitly documented somewhere.